cmd/eskimo/users_test.go

```diff
@@ -0,0 +1,267 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/goccy/go-json"
+	"github.com/ice-blockchain/eskimo/users"
+	. "github.com/ice-blockchain/wintr/testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUsers_Success_Common(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+	defer cancel()
+
+	var allUsers []string
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	ctx = context.WithValue(ctx, "requestingUserIDCtxValueKey", userID) //nolint:staticcheck // No way
+
+	var body string
+	var status int
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("create many users", func() {
+		allUsers = bridge.MustCreateMutuallyReferringUsers(ctx, t)
+	})
+	WHEN("get all users", func() {
+		body, status = bridge.GetUsers(ctx, t, "u", token)
+	})
+	THEN(func() {
+		IT("is successfully returned", func() {
+			assert.Equal(t, 200, status)
+		})
+		IT("verify returned users", func() {
+			verifyMutuallyReferringUsers(t, body, allUsers)
+		})
+	})
+}
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUsers_Failure_RegExp(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), bridge.TestDeadline)
+	defer cancel()
+
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	ctx = context.WithValue(ctx, "requestingUserIDCtxValueKey", userID) //nolint:staticcheck // No way
+
+	var body string
+	var status int
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("empty DB", func() {})
+	WHEN("get all users", func() {
+		body, status = bridge.GetUsers(ctx, t, "u", token)
+	})
+	THEN(func() {
+		IT("is successfully returned", func() {
+			assert.Equal(t, 200, status)
+		})
+		IT("should be empty", func() {
+			assert.Equal(t, body, "[]")
```

Comment:
```suggestion
			assert.Equal(t, "[]", body)
```
**expected** value is the **second** argument, **actual** is the **third**, so is better to change it, otherwise when you get error in the tests it can be difficult to understand what happened.

Reply:
done